app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -98,6 +112,18 @@ class SelectionInteractionViewModel private constructor(
     writtenTranslationContext = translationContext
   }.build()
 
+  override fun getRawUserAnswer(): RawUserAnswer = RawUserAnswer.newBuilder().apply {
+    if (interactionId == "ItemSelectionInput") {
+      itemSelection = ItemSelectionRawAnswer.newBuilder().apply {
+        addAllSelectedIndexes(selectedItems)
+      }.build()
+    } else if (selectedItems.size == 1) {
+      itemSelection = ItemSelectionRawAnswer.newBuilder().apply {
+        addSelectedIndexes(selectedItems.first())
+      }.build()
+    }
```

Comment:
```suggestion
    itemSelection = ItemSelectionRawAnswer.newBuilder().apply {
      if (interactionId == "ItemSelectionInput") {
        addAllSelectedIndexes(selectedItems)
      } else if (selectedItems.size == 1) {
        addSelectedIndexes(selectedItems.first())
      }
    }.build()
```

Slight simplification.

Reply:
Updated